nix/gce.nix

```diff
@@ -412,9 +425,15 @@ in
   config = mkIf (config.deployment.targetEnv == "gce") {
     nixpkgs.system = mkOverride 900 "x86_64-linux";
 
+    # Using NixOs public GCE images by default
+    deployment.gce.bootstrapImage = mkDefault (
```

Comment:
I'm a bit confused by having 2 options `image` `bootstrapImage` do we still need both with this approach ?

Reply:
`image` is an option for the GCEdisk and `bootstrapImage` is an option for GCE deployment
GCE machines will always use one of the bootstrapImages provided in the '*gce-images.nix*' and their corresponding root disks will have that as ***image***

The use case of specifying an *image* from which an extra GCEDisk is created is explained in the previous comment. But it's up to you of course ;)